src/components/Page/components/Header/components/PageTitle/PageTitle.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../../../../styles/common';
+
+.Title {
+  @include text-breakword;
+}
+
+.SubTitle {
+  margin-top: spacing(tight);
+}
+
+.hasThumbnail {
+  display: grid;
+  grid-gap: spacing();
+  grid-template-columns: auto 1fr;
+
+  .TitleAndSubtitleWrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+}
+
+.TitleWithMetadataWrapper {
+  .Title {
+    display: inline;
+    margin-right: spacing(tight);
+
+    // stylelint-disable-next-line selector-max-combinators
+    > * {
```

Comment:
Display block on `DisplayText` needs to `display: inline` if there's meta data

Reply:
I like this a lot. Much better than metadata wrapping to a new line like it does currently. This makes it actually behave like it says it does. 

|  Before | After  |
|---|---|
|<img width="354" alt="metadata-originally-wrapped-to-new-line" src="https://user-images.githubusercontent.com/18447883/61970761-7349de00-afab-11e9-8616-ad74b4e746b0.png">|<img width="357" alt="metadata-only-SMALL" src="https://user-images.githubusercontent.com/18447883/61970760-7349de00-afab-11e9-8971-5e9f129019f4.png">|